digitalpy/core/zmanager/impl/zmq_pusher.py

```diff
@@ -0,0 +1,44 @@
+import zmq
```

Comment:
**[pep8]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>D100 Missing docstring in public module